.github/workflows/pr-to-update-winget.yml

```diff
@@ -0,0 +1,129 @@
+name: PR for updating winget
+
+on:
+  release:
+    types: [released]
```

Comment:
We have 3 types of releases. headlamp, headlamp-plugin, helm.
So this trigger may not be accurate for what we need.
In other similar CI actions we have:
```yaml
  push:
    tags:
    - 'v*'
  workflow_dispatch:
```
The `workflow_dispatch` is for manual triggering when we need it.

Reply:
done!